apps/demo/index.html

```diff
@@ -7,11 +7,13 @@
     <title>H5Web</title>
 
     <meta name="description" content="Web-based HDF5 file viewer." />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/favicon192.png" />
+    <link rel="icon" href="/favicon.ico" />
+    <link rel="apple-touch-icon" href="/favicon192.png" />
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=globalThis"></script>
```

Comment:
Using `polyfill.io` instead of `react-app-polyfill`. Polyfill.io returns a JS file with only the polyfills that are relevant to the browser that makes the request. So in modern browsers, the JS file is empty.

The downside is that the loading of the app is blocked until the polyfill script is loaded. The upside is that we no longer bundle a whole bunch of polyfills that modern browsers don't need. Since this project is a demo, I think it's a good compromise given how simple it is.

Note that the current URL loads Polyfill.io's `default` set of polyfills plus the `gloablThis` polyfill: https://polyfill.io/v3/url-builder/. This is enough for FF 60 to not crash and show the "browser version not supported" message.